src/app/helptext/system/failover.ts

```diff
@@ -23,17 +24,26 @@ export const helptext_system_failover = {
     sync_to_message: T("Sync to peer succeeded."),
   },
 
-  disabled_placeholder: T('Disabled'),
-  disabled_tooltip: T(''),
+  disabled_placeholder: T('Disable Failover'),
+  disabled_tooltip: T('Disables automatic failover and changes the \
+ <i>HA Enabled</i> icon to <i>HA Disabled</i>.'),
 
-  master_placeholder: T('Master'),
-  master_tooltip: T(''),
+  master_placeholder: T(`Default ${globalHelptext.ctrlr}`),
+  master_tooltip: T(`Marks the currently active ${globalHelptext.ctrlr} \
+ as the default active ${globalHelptext.ctrlr} when both \
+ ${globalHelptext.ctrlr}s are online and HA is enabled. To change the \
+ default ${globalHelptext.ctrlr}, unset this option on the default \
+ ${globalHelptext.ctrlr} and allow the system to fail over. This briefly \
+ interrupts system services.`),
 
-  timeout_placeholder: T('Timeout'),
-  timeout_tooltip: T(''),
+  timeout_placeholder: T('Network Timeout Before Initiating Failover'),
+  timeout_tooltip: T('Number of seconds to wait after a network failure \
+ before triggering a failover. <i>0</i> means a failover either occurs \
```

Comment:
"Either" makes it not clear which will happen, but it is deterministic.
```suggestion
 before triggering a failover. <i>0</i> means a failover occurs \
```